python_modules/dagster/dagster/_core/definitions/run_status_sensor_definition.py

```diff
@@ -92,6 +93,9 @@
     Callable[..., RawSensorEvaluationFunctionReturn],
 ]
 
+RUN_STATUS_SENSOR_FETCH_LIMIT = int(os.getenv("DAGSTER_RUN_STATUS_SENSOR_FETCH_LIMIT", "25"))
```

Comment:
depending on how conservative we want to be we could leave this at the default to start

I also wonder if you might want to have either or both of these configurable per sensor?